### PR TITLE
RLS: 2.1.0rc0

### PR DESCRIPTION
**Pull request by lithomas1**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by lithomas1** [comment]
@conda-forge-admin, please rerender

**Comment by lithomas1** [comment]
@conda-forge-admin, please rerender

**Comment by lithomas1** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -62,9 +69,7 @@ test:
     - pytest-cov
     - boto3
     - hypothesis
-    # Workaround recent NumPy 1.24 deprecations, which cause test failures.
-    # xref: https://github.com/conda-forge/pandas-feedstock/issues/150
-    - numpy <1.24
+    - numpy
```
Comment (mroeschke) [inline recipe/meta.yaml:72]: Mind ensuring that some of these packages are synced with https://github.com/pandas-dev/pandas/blob/main/ci/meta.yaml?

**Comment by lithomas1** [comment]
@conda-forge-admin, please rerender

**Comment by lithomas1** [comment]
@conda-forge-admin, please rerender

**Pull request merged**